clear-oily-skin.html

```diff
@@ -114,15 +114,15 @@ <h1>Similar articles</h1>
                 <!-- Flexbox to divide the three recommended articles easier-->
                 <div class="flex-container">
                     <div class="flex-child"><img src="images/acne.png"></div>
-                    <div class="flex-child"><img src="images/moisturising-oily-skin.jpg"></div>
-                    <div class="flex-child"><img src="sample.png"></div>
+                    <div class="flex-child"><img src="images/oily/moisturising-oily-skin.jpg"></div>
+                    <div class="flex-child"><img src="images/dry/help-dry-skin.jpg"></div>

```

Comment:
I added another article in the flex box so now the "similar articles" section is complete for this page as well.